.storybook/config.js

```diff
@@ -0,0 +1,13 @@
+import { configure } from '@storybook/react'
+
+// automatically import all files ending in *.stories.js
+const req = require.context(
+  '../app/javascript/components',
+  true,
+  /\.stories\.js$/
```

Comment:
This changes the default search path for stories to match Webpacker conventions:

`/app/javascript/components/**/*.stories.js`